pyqtgraph/flowchart/library/functions.py

```diff
@@ -327,7 +327,6 @@ def removePeriodic(data, f0=60.0, dt=None, harmonics=10, samples=4):
     
     ## determine frequencies in fft data
     df = 1.0 / (len(data1) * dt)
-    freqs = np.linspace(0.0, (len(ft)-1) * df, len(ft))
```

Comment:
This might be good to keep as a comment, assigning the correct frequencies in FFT is always a pain.

Reply:
I personally hate leaving commented code committed to the repo... we have a lot in this library as it is (many of which have python2 style prints)

[np.fft.fftfreq](https://numpy.org/doc/stable/reference/generated/numpy.fft.fftfreq.html) exists for a reason

Reply:
Fair enough. The following code is also more readable than I was expecting.
Change L 328 to 
`## determine frequency step in fft data`
maybe? You are removing the list of frequencies that the comment now refers to.